Cargo.toml

```diff
@@ -1,3 +1,23 @@
 [workspace]
 
-members = ["xline", "curp", "benchmark", "utils", "engine", "xlineapi", "curp-external-api", "curp-test-utils", "xline-client"]
+members = [
+  "xline",
+  "curp",
+  "benchmark",
+  "utils",
+  "engine",
+  "xlineapi",
+  "curp-external-api",
+  "curp-test-utils",
+  "xline-client",
+  "simulation",
+]
+
+# TODO: We should remove it when the madsim updates its release in the crate.io
```

Comment:
We'd better specify a release number here, otherwise others don't know which release it depends on.

Reply:
done